docs/topics/clusterdefinitions.md

```diff
@@ -895,6 +895,7 @@ https://{keyvaultname}.vault.azure.net:443/secrets/{secretName}/{version}
 | imageReference.gallery        | no       | Name of a Shared Image Gallery.                                                                                                                                                                                                                                 |
 | imageReference.version        | no       | Version of an Image from a Shared Image Gallery.                                                                                                                                                                                                                |
 | sshEnabled                    | no       | If set to `true`, OpenSSH will be installed on windows nodes to allow for ssh remoting. **Only for Windows version 1809/2019 or later**. The same SSH authorized public key(s) will be added from [linuxProfile.ssh.publicKeys](#linuxProfile). Default: `true` |
+| licenseType                   | no       | Specifies that the image or disk that is being used was licensed on-premises. This element is only used for images that contain the Windows Server operating system. <br> Possible values are: Windows_Client, Windows_Server <br> If this element is included in a request for an update, the value must match the initial value. This value cannot be updated. <br> For more information, see [Azure Hybrid Use Benefit for Windows Server](https://docs.microsoft.com/azure/virtual-machines/virtual-machines-windows-hybrid-use-benefit-licensing?toc=%2fazure%2fvirtual-machines%2fwindows%2ftoc.json) <br> Minimum api-version: 2015-06-15 |
```

Comment:
Can you note the default value too? - it looks like that will leave the licensing option empty correct?

Reply:
This is a pointer so the default value is nil and no licenseType will be generated in the template.